pure.zsh

```diff
@@ -47,6 +47,15 @@ prompt_pure_check_cmd_exec_time() {
 	(($elapsed > ${PURE_CMD_MAX_EXEC_TIME:=5})) && prompt_pure_human_time $elapsed
 }
 
+prompt_pure_clear_screen() {
+	# enable output to terminal
+	zle -I
+	# clear screen and move cursor to (0, 0)
+	print -n "\e[2J\e[0;0H"
```

Comment:
Could this use single-quotes? I kinda like to use single-quotes when there's no variable expansion going on to make the intent clear. Does that make sense? (I'm still not sure about it).